idearium-lib/common/redis.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const config = require('./config');
+const Redis = require('ioredis');
+const log = require('./log')('idearium-lib:common/redis');
+
+Redis.Promise.onPossiblyUnhandledRejection(err => log.error({ err }, 'Unhandled Redis error'));
+
+class IORedis {
+
+    constructor () {
+
+        // 10 x 2000ms is about 2 minutes
+        const retryLimit = config.get('redisRetryLimit') || 10;
+        const retryDelay = config.get('redisRetryDelay') || 2000;
+
+        const redis = new Redis(config.get('cacheUrl'), {
+            retryStrategy: (times) => {
+
+                if (times >= retryLimit) {
+
+                    log.fatal('Retry limit reached, could not connect to Redis');
```

Comment:
Could we please put the `redisRetryLimit` value in the message here...

Reply:
@smebberson Updated now.